code/style.css

```diff
@@ -1,7 +1,351 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+html, body {
+  height: 100%;
+  margin: 0;
+  font-family: 'Dosis', sans-serif;
+  background-color: #668483;
+}
+
+/********** HEADER **********/
+
+header {
+  height: 100%;
+  margin-bottom: 100px;
+}
+.hero {
+  background-image: url("./hero-desktop.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 30% 30%;
+  height: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 100px;
+  border-bottom: 2px solid #00000080;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+.hero-text {
+  width: 40%;
+  background-color: #eec29b8f;
+  padding: 40px;
+  margin-right: 50px;
+  box-shadow: 0 0 20px#eec29b;
+}
+h1 {
+  font-family: 'Londrina Solid', cursive;
+  font-size: 4em;
+  margin: 0;
+  text-align: center;
+}
+.hero-text p {
+  font-size: 1.3em;
+  margin-bottom: 0;
+}
+nav {
+  position: relative;
+  background-color: #eec29b;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+nav a {
+  margin: 40px 80px;
+  font-family: 'Londrina Solid', cursive;
+  font-size: 2em;
+  text-decoration: none;
+  color: #000;
+}
```

Comment:
Maybe it would make sense to have the `nav` above the image, at least in mobile - what do you think?